src/ott/solvers/nn/losses.py

```diff
@@ -36,47 +38,78 @@ def monge_gap(
 ) -> Union[float, Tuple[float, sinkhorn.SinkhornOutput]]:
   r"""Monge gap regularizer :cite:`uscidda:23`.
 
-  For a cost function :math:`c` and an empirical reference :math:`\hat{\rho}_n`
-  defined by samples :math:`(x_i)_{i=1,\dots,n}`, the (entropic) Monge gap
-  of a vector field :math:`T` is defined as:
+  For a cost function :math:`c` and empirical reference measure
+  :math:`\hat{\rho}_n` defined by samples :math:`(x_i)_{i=1,\dots,n}`
+  (argument ``reference_points``), the
```

Comment:
I would remove the `(argument ...)` from here (break a bit the flow when reading the docs) and add this description when describe the arguments in `Args:` section.